### PR TITLE
chore: Skipping totp code for org dev

### PR DESCRIPTION
**Pull request by leog**
## What does this PR do?

When working with Org creation, TOTP code will accept any code for lower environments

## Type of change

- [X] Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

Create an org and when TOTP code is asked, enter any code. This only works for non-production environments, such as Local and Dev. QA and PROD will need an actual code.

## Mandatory Tasks

- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

File: packages/lib/constants.ts [hunk 1]
```diff
@@ -34,10 +34,12 @@ export const IS_CALCOM =
   WEBAPP_URL &&
   (new URL(WEBAPP_URL).hostname.endsWith("cal.com") ||
     new URL(WEBAPP_URL).hostname.endsWith("cal.dev") ||
+    new URL(WEBAPP_URL).hostname.endsWith("cal.qa") ||
     new URL(WEBAPP_URL).hostname.endsWith("cal-staging.com"));
 
 export const CONSOLE_URL =
   new URL(WEBAPP_URL).hostname.endsWith(".cal.dev") ||
+  new URL(WEBAPP_URL).hostname.endsWith(".cal.qa") ||
```
Comment (leog) [inline packages/lib/constants.ts:42]: Adding new QA env

File: packages/trpc/server/routers/viewer/organizations/create.handler.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { sendOrganizationEmailVerification } from "@calcom/emails";
 import { hashPassword } from "@calcom/features/auth/lib/hashPassword";
 import { subdomainSuffix } from "@calcom/features/ee/organizations/lib/orgDomains";
 import { DEFAULT_SCHEDULE, getAvailabilityFromSchedule } from "@calcom/lib/availability";
-import { IS_PRODUCTION, IS_TEAM_BILLING_ENABLED, RESERVED_SUBDOMAINS } from "@calcom/lib/constants";
+import { IS_CALCOM, IS_TEAM_BILLING_ENABLED, RESERVED_SUBDOMAINS } from "@calcom/lib/constants";
```
Comment (leog) [inline packages/trpc/server/routers/viewer/organizations/create.handler.ts:8]: Relying on whether the environment is either Dev, QA, or PROD

File: packages/trpc/server/routers/viewer/organizations/create.handler.ts [hunk 3]
```diff
@@ -82,6 +82,8 @@ export const createHandler = async ({ input, ctx }: CreateOptions) => {
   const availability = getAvailabilityFromSchedule(DEFAULT_SCHEDULE);
 
   if (check === false) {
+    if (IS_CALCOM) await vercelCreateDomain(slug);
```
Comment (leog) [inline packages/trpc/server/routers/viewer/organizations/create.handler.ts:85]: Moved up to fail before creating the team if vercel domain creation fails. And relying on whether the environment is either Dev, QA, or PROD

File: packages/trpc/server/routers/viewer/organizations/verifyCode.handler.ts [hunk 4]
```diff
@@ -20,6 +21,8 @@ export const verifyCodeHandler = async ({ ctx, input }: VerifyCodeOptions) => {
 
   if (!user || !email || !code) throw new TRPCError({ code: "BAD_REQUEST" });
 
+  if (!IS_PRODUCTION) return true;
```
Comment (leog) [inline packages/trpc/server/routers/viewer/organizations/verifyCode.handler.ts:24]: If not a PROD environment such as Local or Dev, accept any code.

**Review by keithwillcode — APPROVED** [review]

**Pull request merged**